src/types/type-declarations.ts

```diff
@@ -50,6 +50,15 @@ export interface Inputs {
   [key: string]: any;
 }
 
+export interface Output {
+  id: string;
+  description: string;
+  website?: string;
+  html?: string;
+  stylesheets?: Array<string>;
+  scripts?: Scripts;
+}
+
```

Comment:
Just curious, is the idea that this gets checked by the consumer (i.e Nuxt) to make sure that nothing is missing?

Reply:
The consumer uses our response as an input to some kind of conversion / mapping utility function most likely. 
That's what Nuxt does and I assume something similar happens in Next somewhere.